gtfu/src/main/java/gtfu/Util.java

```diff
@@ -846,4 +846,32 @@ public static String getGTFSURL(AgencyData[] list, String id) {
         return collections;
     }
 
-}
\ No newline at end of file
+    public static Map<String, String> hashMapFromTextFile(String path, String delimiter) {
+
+        Map<String, String> map = new HashMap<String, String>();
+        BufferedReader br = null;
+
+        try {
+            File file = new File(path);
+            br = new BufferedReader(new FileReader(file));
+            String line = null;
+
+            while ((line = br.readLine()) != null) {
+
+                String[] parts = line.split(delimiter);
+                String name = parts[0].trim();
+                String recipients = parts[1].trim();
```

Comment:
This code purports to be a general routine, but these names assume a very specific purpose. Instead of `name` and `recipients`, use something like `key` and `value`

Reply:
Crap. I went back and forth on whether to include in Util, forgot to change the names back :-(